README.md

```diff
@@ -11,16 +16,16 @@ $ npm install crocks -S
 This will pull down `crocks` into your project's `node_modules` folder and can be accessed by adding something like the following in the file that needs it:
 
 ```javascript
-// node require syntax
+// node require syntax - "es5"
```

Comment:
It is not really es5, it is common JS modules.
would rather not make distinctions like this at they become brittle  and stale very quickly.